pecan/routing.py

```diff
@@ -170,18 +173,16 @@ def lookup_controller(obj, remainder, request=None):
 def handle_lookup_traversal(obj, args):
     try:
         result = obj(*args)
+    except TypeError as te:
+        logger.debug('Got exception calling lookup(): %s (%s)',
+                     te, te.args)
+    else:
```

Comment:
I didn't say this wasn't proper Python, and I wasn't trying to make a funny statement. PRs for improving the codebase are good, but can be confusing if the intention of the PR is not clear and it is trying to address other problems that are not specified in the description, which was the case here.